### ISSUE TITLE
Seperation of Filter and Sort

### ISSUE THREAD

**Issue by DJaeger** [jensstein/oandbackup#91]
It is not really clear in the Menu to filter and sort, which items depend to eachother and which are an independing optionset.
There should be some kind of seperation.

**Comment by jensstein** [comment]
yes, i have wanted to make sorting and filtering better for a long time. but it may take a while to get around to it.
